CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+CHANGES IN VERSION 1.10.0 (from 1.9.0)
+===================================
+
+Appium 1.10.0 is a minor release.
+
+#### General
+* Bring Espresso Driver out of beta and into general availability
+* Support Xcode 10, discontinue support for Xcode 8
+* Improve string extractions for Android and iOS
+
+#### iOS
+* Add biometrics (touchId, faceId) [methods](http://appium.io/docs/en/commands/mobile-command/#ios-xcuitest-only) that can enroll/unenroll biometrics and send matching/non-matching biometric inputs
+* Improve video recording (XCUITest)
+* Add possibility to upload files into different container types on Simulator [#770](https://github.com/appium/appium-xcuitest-driver/pull/770)
+* Add [reduceMotion](https://github.com/appium/appium-xcuitest-driver/blob/master/README.md#L161) capability
+
+#### Android
+* Add [mobile:](http://appium.io/docs/en/commands/mobile-command) command for performing editor actions [#428](https://github.com/appium/appium-android-driver/pull/428)
+* Add `localeScript` capability to set script in `locale` (https://developer.android.com/reference/java/util/Locale) [#460](https://github.com/appium/appium-android-driver/pull/460)
+* Improved video recording
+* Fix [bug](https://github.com/appium/appium/issues/11619) calling `getRect` in MJSONWP mode [#240](https://github.com/appium/appium-uiautomator2-driver/pull/240)
+* Fix pin entry error on Samsung devices that wasn't correctly locating the "Enter" button [#458](https://github.com/appium/appium-android-driver/pull/458)
+
```

Comment:
- Start relaxing [hidden api policy](https://developer.android.com/about/versions/pie/restrictions-non-sdk-interfaces) for Android P and over until deleting a session due to handling some hidden API such as changing animation scales.
    - note: Revcent my commit for android, uiautomator2, espresso drivers
- [io.appium.settings](https://github.com/appium/io.appium.settings)
    - Bump the target sdk version to API level 28 so that the settings api can work fully on Android P
    - Initilisation spead improved because of merge IME/Unlock feature into io.appium.settings